docs/getting_started.md

```diff
@@ -0,0 +1,78 @@
+## Getting Started with refine.bio dataset
+
+refine.bio dataset includes gene expression matrices and experiment and sample metadata for the samples that you selected for download.
+
+### Structure
+
+The structure of your download folder is slightly vary depending on whether you chose to aggregate by experiment or by species.  
+
+**The download folder structure for data aggregated by experiment**
+
+![docs-downloads-experiment-agg](https://user-images.githubusercontent.com/15315514/45906716-2f9eaa80-bdc3-11e8-9855-2aaeb74e588d.png)
+
+In this example, two experiments were selected.
+There will be as many folders as there are selected experiments.
+
+**The download folder structure for data aggregated by species**
+
+![docs-downloads-species-agg](https://user-images.githubusercontent.com/15315514/45906715-2f9eaa80-bdc3-11e8-8ab3-90ccc40cfa11.png)
+
+### Contents
+
+* The `aggregated_metadata.json` file contains information about the options you selected for download.
+Specifically, the `aggregate_by` and `scale_by` fields note how the samples are grouped into gene expression matrices and how the gene expression data values were transformed, respectively.
+
+* Individual gene expression matrices and their corresponding sample metadata files are in their own directories.
+
+* Gene expression matrices are the tab-separated value (TSV) files named by the experiment accession number (if aggregated by experiment) or species name (if aggregated by species).
+Note that samples are _columns_ and rows are _genes_ or _features_.
+This pattern is consistent with the input for many programs specifically designed for working with high-throughput gene expression data but may be transposed from what other machine learning libraries are expecting.
+
+* Sample metadata (e.g. disease vs. control labels) are contained in TSV files with `metadata` in the filename as well as any JSON files.
+We apply light harmonization to some sample metadata fields, which are denoted by `refinebio_` (`refinebio_annotations` is an exception).
+The contents of a sample's `refinebio_annotations` field include the submitter-supplied sample metadata.
+
+* Experiment metadata (e.g., experiment title and description) are contained in JSON files with `metadata` in the filename.
+
+Please see the [Downloadable Files](http://docs.refine.bio/en/latest/main_text.html#downloadable-files) section for more details.
```

Comment:
```suggestion
Please see the [Downloadable Files](http://docs.refine.bio/en/latest/main_text.html#downloadable-files) of our documentation section for more details.
```

Maybe? Since this is split off now. Hard for me to visualize what it'll look like/the flow.